packages/components/src/common/styles-dictionary/design-tokens/borders.json

```diff
@@ -0,0 +1,44 @@
+{
+  "sds": {
+    "border": {
+      "negative": {
+        "400": { "value": "1px solid {sds.color.primitive.red.400.value}" }
```

Comment:
The value is coming from [this line](https://github.com/chanzuckerberg/sci-components/blob/6e108696412657e0aed52f2486e4cdcbf00c68cb/packages/components/src/common/styles-dictionary/design-tokens/color.json#L42).

All the design tokens will be merged into one big style-dictionary config because they are all selected [here](https://github.com/chanzuckerberg/sci-components/blob/6e108696412657e0aed52f2486e4cdcbf00c68cb/packages/components/src/common/styles-dictionary/config.js#L48), and we have the option to access them.

Reply:
Ahh I see! So the json file here will still get processed by styles dictionary to produce the final output 💡 Thanks for the education 🚀 !